src/components/pages/index.js

```diff
@@ -22,6 +22,20 @@ function Index({ initialState }) {
   const currentHomeLanguage = homeLanguages[currentHomeLanguageIndex];
   const [isSearching, setIsSearching] = useState(false);
 
+  const handlePageShow = event => {
+    if (event.persisted) {
+      window.location.reload(true);
+    }
+  };
+
+  React.useEffect(() => {
```

Comment:
Why not import with import React, { useState, useEffect } from 'react'?

Reply:
No specific reason, maybe habit, changed to named import as both are the same for me.